Modules/epic.py

```diff
@@ -91,7 +91,7 @@ def __eq__(self, other: 'Epic') -> bool:
         # while it is bad style to check a value against None, its possible this object
         # will be None inside the rescue object, for rescues that are not Epics.
         if other is None:
-            return False
+            return NotImplemented
 
         # type check
         if not isinstance(other, Epic):
```

Comment:
If I read the contract correctly, here is where you'd return `NotImplemented` rather than if other is `None`.

Reply:
... here i was thinking i fixed all that. yes, the None check is redundant as it would be caught by the `isinstance` check, which would also need to return `NotImplemented` rather than a raised type error.

Guess i missed  a spot